README.md

```diff
@@ -146,7 +146,7 @@ This takes you through the process of creating a file called a `package.json` wh
   "engines": {
     "node": ">= 0.10"
   },
-  "dependedncies": {
+  "dependencies": {
      "pre-commit": "^1.0.7"
```

Comment:
That would be more precise, but we haven't gone into explaining the difference between dependencies and devDependencies in the readme. Do you think that is needed at this stage?